guides/user/plugin-users/index.md

```diff
@@ -11,78 +11,191 @@ sidebar:
   <strong>Note:</strong> Plugins are an early alpha feature that is under active development and will likely change.
 </div>
 
-This guide is for adding existing plugins to Spinnaker. For information about how to develop a new plugin, see [Plugin Creators Guide](/guides/developer/plugin-creators). 
+In this guide, you add an existing plugin from an [example repository](https://github.com/spinnaker-plugin-examples/examplePluginRepository) to Spinnaker. See the [Plugin Creators Guide](/guides/developer/plugin-creators) for how to create a PF4J or Spring plugin.
+
+# Requirements
+
+* The plugin is either a PF4J or a Spring plugin
+* The plugin resides in a publicly accessible place
+* Spinnaker v1.19 or later
+* Halyard v1.32.0 or later
+* You redeploy Spinnaker using Halyard to apply changes
+
+# Define Plugins
+
+Define plugins in a file called `plugins.json`. This guide uses the  [file](https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json) in the example repository.
```

Comment:
I think the Markdown interpreter ignores the extra space?

Reply:
@dorbin I reviewed the changes locally, but I can't say for sure that I did or did not notice 2 spaces in the rendered output. 
 Is spinnaker.github.io deployed using Netlify? It would be nice to have preview builds of each PR.

Reply:
I built it locally and checked. It does only render 1 space. That's good to know.

Reply:
I found and removed the extra space :)